rust/.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: rust
+rust:
+  - nightly
```

Comment:
Do we need to be on Rust nightly? may be wishful thinking but I was hoping after almost 30 releases people would be using stable ;)

Reply:
> people would be using stable

Ha, good one! :)

Unfortunately, this crate requires `#![feature(allocator_api, box_syntax, fn_traits, try_from, unboxed_closures)]` (`box_syntax` is _totally_ necessary).